### PR TITLE
Remove nft detail from nft collection data

### PR DESCRIPTION
**Pull request by victorkirov**
# 🔘 PR Type

What kind of change does this PR introduce?

- [x] Enhancement

# 📜 Background

We were fetching too much NFT info in our collection query. This separates concerns a bit.

# 🔄 Changes

Does this PR introduce a breaking change?

- [x] Yes, Incompatible API changes
- [ ] No, Adds functionality (backwards compatible)
- [ ] No, Bug fixes (backwards compatible)

Changes:

- NFT detail data is removed form colleciton data

Impact:

- Will need a rewrite of the NFT collection screens on mobile and extension

# ✅ Review checklist

Please ensure the following are true before merging:

- [ ] Code Style is consistent with the project guidelines.
- [ ] Code is readable and well-commented.
- [ ] No unnecessary or debugging code has been added.
- [ ] Security considerations have been taken into account.
- [ ] The change has been manually tested and works as expected.
- [ ] Breaking changes and their impacts have been considered and documented.
- [ ] Code does not introduce new technical debt or issues.

### CODE REVIEW

File: stacksCollectible/index.ts [hunk 1]
```diff
@@ -69,43 +63,63 @@ export async function getAllNftContracts(
...
 
-  batch.forEach((nft) => {
+  for (const nft of nfts) {
     const principal: string[] = nft.asset_identifier?.split('::');
     const contractInfo: string[] = principal[0]?.split('.');
     const contractId = principal[0];
 
+    const tokenId = nft.value.repr.replace('u', '');
+    const contractName = contractInfo[0];
+    const contractAddress = contractInfo[1];
```
Comment (fedeerbes) [inline stacksCollectible/index.ts:81]: question: shouldn't contractName be in position 1 and contractAddress in position 0?

Comment (victorkirov) [reply stacksCollectible/index.ts:81]: Oooops 🙈 Good catch

**Comment by teebszet** [comment]
@victorkirov are there duplicates in the returned response?

image

Screenshot 2023-11-17 at 11 28 47 AM

**Comment by victorkirov** [comment]
> @victorkirov are there duplicates in the returned response?
> 
> image
> 
> Screenshot 2023-11-17 at 11 28 47 AM

Weird! Will look into it. What stx address were you using?

**Review by teebszet — APPROVED** [review]

**Pull request merged**